ui/src/pages/backoffice/Patron/PatronDetails/components/PatronDocumentRequests/PatronDocumentRequests.js

```diff
@@ -30,24 +30,23 @@ export default class PatronDocumentRequests extends Component {
 
   viewDetails = ({ row }) => {
     return (
-      <Button
-        as={Link}
+      <Link
         to={BackOfficeRoutes.documentRequestDetailsFor(row.metadata.pid)}
-        compact
-        icon="info"
         data-test={row.metadata.pid}
-      />
+      >
+        {row.metadata.pid}
+      </Link>
     );
   };
 
   render() {
     const { data, isLoading, error } = this.props;
     const columns = [
-      { title: '', field: '', formatter: this.viewDetails },
-      { title: 'ID', field: 'metadata.pid' },
+      { title: 'ID', field: 'metadata.pid', formatter: this.viewDetails },
       { title: 'Document ID', field: 'metadata.document_pid' },
+      { title: 'Document title', field: 'metadata.title' },
       { title: 'State', field: 'metadata.state' },
-      { title: 'Created', field: 'created', formatter: dateFormatter },
+      { title: 'Created on ', field: 'created', formatter: dateFormatter },
```

Comment:
same here

Reply:
in case of the dateFormatter it uses the field, so I can't remove it